covid-dashboard/src/js/modules/map.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-undef */
+import 'leaflet/dist/leaflet-src.js';
+import 'leaflet.bigimage';
+import mapAddMarkers from './mapAddMarkers.js';
+import mapLegendInit from './mapLegendInit.js';
+import mapMenuInit from './mapMenuInit.js';
+import mapFullscreenInit from './mapFullscreenInit.js';
+import dom from './dom.js';
+
+export default () => {
```

Comment:
[Optional] It's better to limit zooming out to avoid seeing multiple Earths.
Also as there is no data by locations there is no need in very detailed zooming in.

Reply:
fixed

Reply:
It's still zooming in and out very much. Did you deploy it?

Reply:
deploy here: https://rolling-scopes-school.github.io/helias17-JS2020Q3/covid-dashboard/